src/main/java/org/openstreetmap/atlas/checks/validation/tag/FixMeReviewCheck.java

```diff
@@ -0,0 +1,160 @@
+package org.openstreetmap.atlas.checks.validation.tag;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.openstreetmap.atlas.checks.base.BaseCheck;
+import org.openstreetmap.atlas.checks.flag.CheckFlag;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasObject;
+import org.openstreetmap.atlas.geography.atlas.items.Edge;
+import org.openstreetmap.atlas.geography.atlas.walker.OsmWayWalker;
+import org.openstreetmap.atlas.tags.BuildingTag;
+import org.openstreetmap.atlas.tags.HighwayTag;
+import org.openstreetmap.atlas.tags.PlaceTag;
+import org.openstreetmap.atlas.tags.SurfaceTag;
+import org.openstreetmap.atlas.tags.WaterwayTag;
+import org.openstreetmap.atlas.tags.names.NameTag;
+import org.openstreetmap.atlas.tags.names.ReferenceTag;
+import org.openstreetmap.atlas.tags.oneway.OneWayTag;
+import org.openstreetmap.atlas.utilities.configuration.Configuration;
+
+/**
+ * This check flags features which contain the "fixme" or "FIXME" tag as well as other combinations
+ * of important tags.
+ *
+ * @author v-garei
+ */
+public class FixMeReviewCheck extends BaseCheck<Long>
+{
+    private static final long serialVersionUID = -2191776278923026805L;
+    private static final String HAS_FIXME_TAG = "Object {0, number, #} has 'fixme' tag and needs to be investigated.";
+    private static final List<String> FALLBACK_INSTRUCTIONS = Collections
+            .singletonList(HAS_FIXME_TAG);
+    private static final List<String> FIX_ME_SUPPLEMENTARY_TAGS = List.of(WaterwayTag.KEY,
```

Comment:
if this is expected to ever change, it could be configurable

Reply:
yeah agreed. I will change this.

Reply:
For now this is not changing. I will have another PR for this check once I update the Atlas repo. I think this is okay for now.